libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -751,6 +751,10 @@ void SRV_Channels::set_slew_rate(SRV_Channel::Aux_servo_function_t function, flo
     for (slew_list *slew = _slew; slew; slew = slew->next) {
         if (slew->func == function) {
             // found existing item, update max change
+            if (!is_positive(slew->max_change)) {
+                // prevent a step when enabling/disabling
+                slew->last_scaled_output = functions[function].output_scaled;
+            }
```

Comment:
I cant work out why we need this? `last_scaled_output` should always be `output_scaled` when max_change is zero?

https://github.com/ArduPilot/ardupilot/blob/e9942f9a17ab87a7e67821f25c997330d9286f61/libraries/SRV_Channel/SRV_Channels.cpp#L437-L441